classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
```

Comment:
- [ ] in order for this relationship to work the student instance variable should be readable.